sources/src/main/java/com/ullink/slack/simpleslackapi/impl/SlackWebSocketSessionImpl.java

```diff
@@ -400,51 +400,47 @@ private void establishWebsocketConnection() throws IOException
         }
         establishConnectionThread = null;
 
-        establishConnectionThread = new Thread()
+        establishConnectionThread = new Thread(() ->
```

Comment:
You shouldn't use lambdas or any other java 8 feature, the source and bytecode should remain compatible with java 7 for now.